packages/styled-react/src/__tests__/primer-react.browser.test.tsx

```diff
@@ -114,16 +113,6 @@ describe('@primer/react', () => {
     expect(window.getComputedStyle(screen.getByTestId('component')).backgroundColor).toBe('rgb(255, 0, 0)')
   })
 
-  test('Breadcrumbs supports `sx` prop', () => {
-    render(<Breadcrumbs sx={{background: 'red'}} />)
-    expect(window.getComputedStyle(screen.getByLabelText('Breadcrumbs')).backgroundColor).toBe('rgb(255, 0, 0)')
-  })
-
-  test('Breadcrumbs.Item supports `sx` prop', () => {
-    render(<Breadcrumbs.Item data-testid="component" sx={{background: 'red'}} />)
-    expect(window.getComputedStyle(screen.getByTestId('component')).backgroundColor).toBe('rgb(255, 0, 0)')
-  })
-
```

Comment:
But what does this mean to all the usages of `Breadcrumbs` with `sx` prop?
https://primer-query.githubapp.com/?query=name%3ABreadcrumbs+attribute%3Asx+repo%3Agithub%2Fgithub-ui